Notely/lib/screens/settings.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/material.dart';
+import 'package:settings_ui/settings_ui.dart'; //Flutter Settings Package
+//Added under 'dependencies' in pubspec.yaml
+
+List<Icon> icons = [
+  Icon(Icons.person),
+  Icon(Icons.settings),
+  Icon(Icons.credit_card),
+];
+
+Size buttonSize;
+Offset buttonPosition;
+bool isMenuOpen = false;
+
+class SettingsPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    bool privateAccountValue = false;
+    String dropdownValue = '';
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Settings Page'),
+        elevation: 1,
+        leading: IconButton(
+          icon: Icon(
+            Icons.arrow_back,
+            color: Colors.white,
+          ),
+          onPressed: () {
+            Navigator.pop(context);
```

Comment:
I believe that when using navigator to open a new screen, a "back arrow" is provided by default